### ISSUE TITLE
Never used variable turns off autocomplete

### ISSUE THREAD

**Issue by yisonPylkita** [dlang-vscode/dlang-vscode#10]
With this code autocomplete works fine:

``` D
import deimos.ncurses;

void main()
{
    deimos.ncurses.initscr();
    deimos.ncurses.endwin();  /// Stil works
}
```

But with this not:

``` D
import deimos.ncurses;
alias NC = deimos.ncurses;

void main()
{
    deimos.ncurses.initscr();
    immutable hello = toStringz("Hello World");
    deimos.ncurses.endwin();  /// Never used varable hello turns of autocomplete
}
```

**Comment by LaurentTreguier** [comment]
This is weird, I can't reproduce this. I still get completions after the unused variable... Is it at `deimos.ncurses.` that completions are missing ?

**Comment by yisonPylkita** [comment]
Thanks for reply.
You are right, code above works. I little modified code I had placed here, so you could not reproduce bug. Sorry, my bad.

The issue occurs when 

``` D
immutable message = toStringz("Hello World!");
```

is replaced with:

``` D
immutable message = toStringz("Hello World!日本語からの");
```

With second one autocomplete does not work

This is not caused by that `import deimos.ncourses`.  I have created new project:

_test.d_

``` D
module test;

class Test {

}
```

_app.d_

``` D
import std.stdio;
import test;

void main()
{
    immutable msg = "Hello World!日本語からの";
    writeln(msg);  /// Here autocomplete does not work
        /// Everything below also does not has autocomplete
}
```

and I can still reproduce this bug

**Comment by LaurentTreguier** [comment]
Now I can reproduce it too. But I don't get it. DCD expects its input to be UTF-8 encoded, VSCode defaults to UTF-8 for its files, and I tried forcing the extension to send UTF-8 encoded text to DCD, and nothing changes.
It's the same with the autocomplete package I had done for Atom, having non-ASCII characters seem to mess up DCD.

I've opened an issue [here](https://github.com/Hackerpilot/DCD/issues/334) about this problem
